mmedit/models/losses/perceptual_loss.py

```diff
@@ -136,12 +140,25 @@ def __init__(self,
         self.perceptual_weight = perceptual_weight
         self.style_weight = style_weight
         self.layer_weights = layer_weights
+        self.layer_weights_style = layer_weights_style
```

Comment:
This line is not necessary. In line 151 we can use `layer_weights_style` instead of `self.layer_weights_style`.

Reply:
Hi @ckkelvinchan sorry for the late reply! I have just been assigned to a different project so have been rather busy with it. 
Since `layer_weights_style` is also called in the `forward` function, I think keeping it as `self.layer_weights_style` may make life easier. As you can see in the `forward` function, `layer_weights_style` becomes undefined when `self` is removed from `__init__()`

<img width="996" alt="image" src="https://user-images.githubusercontent.com/33197366/160246747-ce4f63d9-4f62-4fe0-b2e0-215735a0241c.png">

Reply:
Don’t be sorry, and we greatly appreciate your help!

I missed that part and yes we should keep this. Thanks for your reminder. In this case I think there is only one minor point (as mentioned above) and we are good to go for the unittest. Really thanks for your help!

Reply:
It's really been a while haha. Cool let me write up the unit tests asap! Thanks kelvin!